docker-compose.yml

```diff
@@ -31,3 +31,59 @@ services:
     depends_on:
       db:
         condition: service_healthy
+  # Sentry setup
+  sentry:
+    image: sentry:8.17.0
+    environment:
+      # Run `docker-compose run web config generate-secret-key`
```

Comment:
Could you expand on this?
If I run this command to generate a secret key, why is it set in the line below?